AK/Statistics.h

```diff
@@ -29,6 +29,28 @@ requires(IsArithmetic<T>) class Statistics {
     T const sum() const { return m_sum; }
     float average() const { return (float)sum() / size(); }
 
+    T const min()
```

Comment:
Returning `T const` here is weird, this should just be `T`

Reply:
...I see that the entirety of the file is using `T const`, since it's not particularly significant (`T` most likely fits in a register, so the copy caused by it is essentially free and the compiler can most likely see through it anyway), we can keep it like this and address it later.

However, please mark these functions themselves `const`.

Reply:
This could be addressed in the "Little Tweaks" PR we talked about on #9750